test/e2e/helpers.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eu
+
+BIN="${BIN:-./_output/shp}"
+
+function die () {
+	echo $* >&2
+	exit 1
+}
```

Comment:
"die" might be considered exclusionary language, in the same category as "kill." I would replace this with "fail"
```suggestion
function fail () {
	echo $* >&2
	exit 1
}
```